### DISCUSSION TITLE
Suddenly workflows aren't getting triggered on push

### DISCUSSION THREAD

**Discussion by samsonitto** [community/community#25476] [Actions]
<p>Two hours ago everything worked fine. Now all of a sudden workflows are not being triggered on the push to the master branch. I tried it with two different repos. I did not edit my action scripts. Very frustrating. Some help?</p>

**Comment by rmorshea** [comment]
<p>Experiencing the same: <a href="https://github.com/idom-team/idom" rel="noopener nofollow ugc">https://github.com/idom-team/idom</a></p>

**Comment by Grantimus9** [comment, marked as answer]
<p>Most likely will be resolved with resolution of this: <a href="https://www.githubstatus.com/incidents/28js6274ybpb" rel="noopener nofollow ugc">https://www.githubstatus.com/incidents/28js6274ybpb</a></p>
